app/Console/ConsoleServiceProvider.php

```diff
@@ -514,7 +514,8 @@ function () use ($container) {
                     LoggerFactory::create(
                         $container,
                         LoggerName::forService('fetching-movies', 'kinepolis')
-                    )
+                    ),
+                    $container->get('config')['kinepolis']['trailers']['enabled'],
```

Comment:
The list of parameters is getting rather big. This makes me wonder if it should become part of `YoutubeTrailerRepository`